hack/check-generated-manifests.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ibm_operator_name=ibm-block-csi-operator
+cd ..
+cp -r $ibm_operator_name ./$ibm_operator_name-copy
+cd $ibm_operator_name-copy/
+hack/update-crds.sh
```

Comment:
why not update all?

Reply:
what do you meen? we are performing generation to all of the yaml files

Reply:
```suggestion
hack/update-all.sh
```

Reply:
but we wanted to check only the 
`make generate` and `make manifest` commands no? because we are not really using `hack/update-installer.sh` script.

Reply:
I think we should manage all yamls in the same way.
could we try to resume using `hack/update-installer.sh` as well?
it might require some alignment to the new dirs structure

Reply:
I agree that we need to use it, but we didn't use it before this PR, so I think its something for another ticket

Reply:
did we use `hack/update-crds.sh` before this PR?

Reply:
I think @zingero  used it when we upgraded the operator-sdk / upgrading the crd kind

Reply:
It looks familiar

Reply:
I don't say that we need to treat them differently, I just stayed we need to do it in another ticket, because it's not related to this PR

Reply:
ok, then could you open a ticket for it and write its number here?

Reply:
CSI-3252